packages/visualizations/src/components/Table/Pagination/Pagination.svelte

```diff
@@ -40,7 +40,9 @@
 <style lang="scss">
     .pagination {
         display: flex;
-        justify-content: space-between;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        gap: var(--spacing-100);
```

Comment:
Better for vertical display indeed, looks a bit strange with the spacer but it will look good with the page size indeed 👌